package.json

```diff
@@ -71,6 +71,12 @@
                     "when": "view == zeplin.views.barrels"
                 }
             ],
+            "view/item/context": [
+                {
+                    "command": "zeplin.sidebar.removeBarrel",
+                    "when": "view == zeplin.views.barrels && viewItem == barrel"
```

Comment:
Can `viewItem` have another value other than `barrel` if view is `zeplin.view.barrels`?

Reply:
"barrels" is one of the sidebar headers.
"barrel" is one of the views in the sidebar.
In "barrels" there will also be "screens", "sections", "components" etc.

Reply:
Gotcha!